svc/pkg/user-identity/ops/get/src/lib.rs

```diff
@@ -2,20 +2,35 @@ use proto::backend::{self, pkg::*};
 use rivet_operation::prelude::*;
 
 #[derive(Debug, sqlx::FromRow)]
-struct IdentityRow {
+struct EmailRow {
 	user_id: Uuid,
-	email: Option<String>,
+	email: String,
 }
 
-impl From<IdentityRow> for user_identity::get::CacheUserIdentity {
-	fn from(val: IdentityRow) -> Self {
-		user_identity::get::CacheUserIdentity {
+impl From<EmailRow> for user_identity::get::CacheUserEmailIdentity {
+	fn from(val: EmailRow) -> Self {
+		user_identity::get::CacheUserEmailIdentity {
 			user_id: Some(val.user_id.into()),
 			email: val.email,
 		}
 	}
 }
 
+#[derive(Debug, sqlx::FromRow)]
+struct AccessTokenRow {
+	user_id: Uuid,
+	name: String,
+}
+
+impl From<AccessTokenRow> for user_identity::get::CacheUserAccessTokenIdentity {
+	fn from(val: AccessTokenRow) -> Self {
+		user_identity::get::CacheUserAccessTokenIdentity {
+			user_id: Some(val.user_id.into()),
+			name: val.name,
+		}
+	}
+}
+
 #[operation(name = "user-identity-get")]
 async fn handle(
```

Comment:
We're running two queries in parallel here. I think they can be combined to be faster & simpler with a `UNION`. Something like

```
SELECT user_id, email, NULL AS access_token_name
FROM db_user_identity.emails
WHERE user_id = ANY($1)
UNION ALL
SELECT user_id, NULL AS email, name AS access_token_name
FROM db_user_identity.access_tokens
WHERE user_id = ANY($1)
```

Can be done with `LEFT JOIN` too.